rust/theoros/src/rpc/evm/mod.rs

```diff
@@ -0,0 +1,43 @@
+pub mod hyperlane;
+
+pub use hyperlane::*;
+
+use std::collections::HashMap;
+
+use alloy::hex::FromHex;
+use alloy::primitives::Address;
+use url::Url;
+
+use crate::configs::evm_config::{EvmChainName, EvmConfig};
+
+#[derive(Debug, Clone)]
+pub struct HyperlaneRpcs(HashMap<EvmChainName, HyperlaneClient>);
+
+impl HyperlaneRpcs {
```

Comment:
HyperlaneClientsMapping

Reply:
And maybe we should just implement the from trait directly ?

Reply:
Used `HyperlaneRpcsMapping` to have use the same wording for stuff in the `rpc` module

Reply:
> And maybe we should just implement the from trait directly ?

not that straight/clear since it needs to be async + propagate errors, so clearer to just do it like that imo

Reply:
Renamed to `HyperlaneValidatorsMapping` and stored the validators directly